### ISSUE TITLE
Pretrained model and unavailability of data to replicate results

### ISSUE THREAD

**Issue by ajinkyakulkarni14** [yochaiye/scene_agnostic_dereverberation#1]
Hello, THank you for release of the code. Will it be possible to release the pretrained model?

Also, as the WSJ0 dataset is not free available, it is difficult to run the code. Will it be possible to state the shape of input tensor and output tensor? 

Thank you.

**Comment by yochaiye** [comment]
@ajinkyakulkarni14 
Hi,
Sorry for only getting back to you now!

I will be able to upload the trained model in the next few days.
Any preference of microphones number?
Noisy/Noiseless dataset?

As for the second question - unfortunately you must have the WSJ0 in order to create the dataset.
Anyway, the input shape is BATCH_SIZE x MICS_NUM x 256 x 256 x 1, and the output shape is BATCH_SIZE x 256 x 256 x 1

**Comment by ajinkyakulkarni14** [comment]
Thank you so much for your reply. I am working with three microphones with Noisy dataset in Far field scenario.

Also, can you also explain input shape and output shape, is it batch_size x mics_num x frame_len x feat_dim x 1 or  batch_size x mics_num  x feat_dim x frame_len x 1??

**Comment by yochaiye** [comment]
From what I see, the format at the input is batch_size x mics_num x time x frequency.
I will initiate a model training a model and then upload it.
By the way, if 4 microphones is ok for you, I have an off-the-shelf model to upload.
Let me know

**Comment by yochaiye** [comment]
@ajinkyakulkarni14 
I trained a model, but it is too large to upload.
Have you got a google drive account?

**Comment by ajinkyakulkarni14** [comment]
@yochaiye Yes, I have google drive. Also, Can you share your email id? My email id is ajinkyakulkarni14@gmail.com

**Comment by yochaiye** [comment]
@ajinkyakulkarni14 
I have sent you a link, you can check your inbox.

**Comment by ajinkyakulkarni14** [comment]
@yochaiye I have received the model thanks for the help.
